python/lsst/rubintv/production/rubinTv.py

```diff
@@ -1386,8 +1473,9 @@ def run(self):
 
                 # TODO: need to work out a better way of dealing with pod
                 # restarts. At present this will just remake everything.
-                justMade = self.processDay(dayObs, skipEventNumbers=plotted)
-                plotted.update(justMade)
+                mountMotionJustPlotted, icsJustPlotted = self.processDay(dayObs, skipEventNumbers=plotted)
+                mountMotionPlotted.update(mountMotionJustPlotted)
```

Comment:
Do these sets get used anywhere?  Something certainly should replace `plotted` on line 1476, but I'm not sure if it should be one of these, or the union of the two.

Reply:
Oh, that's a great catch, thank you. And uggg, you're right, technically it's neither - this is why it needs refactoring. Will take a look into that now.